recipe/meta.yaml

```diff
@@ -25,25 +24,30 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
+    # This basically forces to pick scipy (and any other suitable packages in the chain, like numpy) 
+    # that was built with openblas rather than mkl. Openblas typically pairs with llvm-openmp and mkl with intel-openmp on osx-64. 
+    # By default, mkl variants are picked. By picking the variant that uses openblas rather 
+    # than mkl, intel-openmp will not be pulled in and so you will not have llvm-openmp / intel-openmp conflict. 
+    - blas=*=openblas # [osx and x86_64]
+    - scikit-build-core >=0.4.4
     - python
     - pip
     - setuptools
     - wheel
-    - llvm-openmp 14.0.6  # [osx]
-    # libgomp pulled in through gcc setup
-    # - libgomp 11.2.0      # [linux]
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
   run:
+    - blas=*=openblas # [osx and x86_64]  
     - python
     - numpy
     - scipy
-    - scikit-learn !=0.22.0
-    - _openmp_mutex      # [linux]
 
 test:
   imports:
     - lightgbm
   requires:
     - pip
+    - scikit-learn !=0.22.0
```

Comment:
```suggestion
    - scikit-learn
```

Reply:
ah got it, it will be constrained in the `run_constrained`. Thank you